design-system/docs/components/sidebar/sidebar.tsx

```diff
@@ -15,67 +16,55 @@ import { ColorSchemeMenu } from '../theme-switcher';
 
 /** Responsively render sidebar navigation items. */
 export const Sidebar = ({ items }: { items: SidebarItem[] }) => {
-  // const [headerHeight, setHeaderHeight] = useState(NaN);
   const { sidebarIsOpen, closeSidebar, toggleSidebar } = useSidebarContext();
+  const pathname = usePathname();
 
   useEffect(() => {
-    addEventListener('popstate', closeSidebar);
```

Comment:
This wasn't working to close the menu 

**Current experience:**
https://github.com/user-attachments/assets/befa47e4-bfa5-42ef-a220-c5466045c36b

**New experience:**
https://github.com/user-attachments/assets/7b6bc1f6-da71-4499-892f-2b6615787b04